pkg/v11/resource/tiller/delete.go

```diff
@@ -2,20 +2,10 @@ package tiller
 
 import (
 	"context"
-
-	"github.com/giantswarm/microerror"
 )
 
+// EnsureDeleted is not implemented for the tiller resource. Tiller will be
+// deleted when the tenant cluster resources are deleted.
```

Comment:
For tiller we don't want to delete it because it goes away with the tenant cluster.